src/DotVVM.Framework/ResourceManagement/ScriptResource.cs

```diff
@@ -44,5 +51,23 @@ public void RenderPreloadLink(IHtmlWriter writer, IDotvvmRequestContext context,
             writer.RenderBeginTag("link");
             writer.RenderEndTag();
         }
+        protected override void RenderFallbackLoadingScript(IHtmlWriter writer, IDotvvmRequestContext context, string resourceName, IResourceLocation fallback, string javascriptCondition)
+        {
+            var text = new StringWriter();
+            var hw = new HtmlWriter(text, context);
+            RenderLink(fallback, hw, context, resourceName, false);
```

Comment:
No blocker, but actually, is it required to exclude the defer attribute from the fallback script? As the attributes are copied into a new script tag without the `defer` attribute, it should have no effect, right?